src/test/kotlin/GeneratedTests.kt

```diff
@@ -259,5 +259,10 @@ class GeneratedTests {
     fun testCompositeTypesLsl() {
         runLslTest("compositeTypes")
     }
+
+    @Test
+    fun testArrayListTypeScriptLsl() {
+        // runLslTest("ArrayList.main")
+    }
```

Comment:
I think it's not a good practice to reference external code-bases with unknown/unclear public access status even for testing without explicit agreement.
On the other hand it's good to validate from time to time improvements and backward-compatibility with existing code/specs.
I strongly suggest using some other means of validation on external/third-party code-bases (such as making a special app/micro-project to run parser locally).

Reply:
@dog-m I agree with you absolutely ) 

I have tested this spec and have forgotten to delete this lines (

Reply:
Fixed;